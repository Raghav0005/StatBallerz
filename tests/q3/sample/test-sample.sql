DELETE FROM User u WHERE u.username = 'user1'
