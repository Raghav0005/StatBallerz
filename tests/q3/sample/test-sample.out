
DB21007E  End of file reached while reading the command.
